configuration/persistence.md

```diff
@@ -7,12 +7,238 @@ title: Persistence
+A complete list of supported persistence add-ons may be found in the [persistence]({{base}}/addons/persistence.html) section of the on-line openHAB documentation.
+
+## Persistence Add-on Configuration
+
+Each persistence add-on you install will need to be configured.
+Please refer to the specific [on-line documentation]({{base}}/addons/persistence.html) for your selected persistence add-on for configuration instructions.
+
+## Default Persistence Service
+
+You may install more than one persistence add-on.
+Therefore, it is important to select a default persistence service.
+You should do this even if you have only one persistence add-on installed.
+
+To select a default persistence service, in paperUI, select Configuration and then System from the side menu.
+Scroll down to "Persistence", and select your Default Service from the drop-down list.
+Note that you must first install a persistence add-on before you make this selection.
+Be sure to save your choice once you have selected your default service.
+
+## Configuration
+
+You can choose how your persistence service operates, which item states it persists, and under what conditions it persists them.
+These things and more are configured in a file named `<persistenceservice>.persist`, where "persistenceservice" is replaced by the name of your add-on (e.g. `rrd4j.persist`).
+This folder is located in `$OPENHAB_CONF/persistence`.
+The configuration defines persistence "strategies" which are very similar to [triggers]({{base}}/configuration/rules-dsl.html#rule-triggers) in openHAB rules.
+These strategies may be used to persist an item state when some bus event has occurred (e.g. an item state has been updated or changed), or on a schedule or at a specific time of day (e.g. through a [cron expression](http://www.quartz-scheduler.org/documentation/quartz-2.1.x/tutorials/crontrigger).
+
+Persistence configuration files consist of several sections:
+
+### Strategies section
+
+This section allows you to define strategies and to declare a set of default strategies to use for this persistence service.  The syntax is as follows:
+
+```java
+Strategies {
+  <strategyName1> : "cronexpression1>"
+  <strategyName2> : "cronexpression2>"
+  ...
+
+  default = <strategyNameX>, <strategyNameY>
+}
+```
+
+Note that the `Strategies` section must be included, and that it must define a default, or the persistence service will not work.
+
+The following strategies are defined internally in openHAB for your convenience:
+
+- everyChange: persist the item state whenever its state has changed
+- everyUpdate: persist the item state whenever its state has been updated, even if it did not change
+- restoreOnStartup: load and initialize the last persisted item state on openHAB startup (if the item state is undefined (`UNDEF`)).  This strategy is extremely useful for items that are not updated through a Thing with a Binding to something in the real world.
```

Comment:
Add "see below" instead of the extra sentence?

Reply:
Definitely.